### DISCUSSION TITLE
Is there any way to run 2 AGH in a kind of "master-slave" configuration?

### DISCUSSION THREAD

**Discussion by 71z0** [AdguardTeam/AdGuardHome#4596] [Q&A] [answered]
I run 2 AGH as DNS, one inside Home Assistant on a Raspberry Pi, the other alone on a second Raspberry Pi.
Keeping both manually in an identical configuration is extremely time consuming and annoying.
Isn't there a way to configure the second one - no matter which of the two instances is the first or the second - so that it pulls the current configuration of the first one at startup?

This way you would only have to configure one via the interface and then restart the second.

**Comment by ainar-g** [comment, marked as answer]
Not currently.  But see #573 for some recipes and the future plans regarding the problem.

**Reply by 71z0** [reply]
Thanks a lot, did not find this before :-/
So, we're waiting for the v0.108.0 milestone for an easier way to realize redundancy.
